### ISSUE TITLE
[a11y]: ai-chat should provide status message before and after clearing chat history

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#881]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

Clear chat status Messages are not in place

1. Before clearing the chat history , there is no second confirmation to user.
2. After clearing the chat , there is no status updates to users.

Recommended fix: For the benefits of Assistive Aid users:

1. Initiate a Second confirmation Mechanism before clearing the existing Chat, for example : an Are you sure ? Do you needs to clear the existing Chat history ?
2. Provide a Sasses message to indicate the Action completions after Chat clearing, for example :Your Chat history successfully cleared ! Ok Button

**Note:** Both Messages must be instantly announced by screen reader and should provide proper Keyboard focus /access to make it accessible for Keyboard /screen reader users.

User impact: Without confirmation before clearing and no status message after, screen reader users may lose chat history by mistake and receive no feedback that the action was completed, causing confusion and uncertainty.

### WCAG 2.1 Violation

WCAG Mapping: 4.1.3 Status Messages

### Reproduction/example

n/a

### Steps to reproduce

1. make sure JAWS is active
2. open ai-chat
3. navigate to burger menu at the top left of ai.chat window
4. select clear history from the menu
5. see that there is no message before clearing the message history
6. see that there is no message after clearing the message history

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
